tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -175,10 +175,10 @@ module.exports = {
      */
     restoreFile: function (fileName) {
       return this.initAjaxCounters()
-        .waitForFileVisible(fileName)
+        .waitForFileWithPathVisible(fileName)
         .useXpath()
         .performFileAction(fileName, 'restore')
-        .waitForElementNotPresent(this.getFileRowSelectorByFileName(fileName))
```

Comment:
I think we should add an `waitForOutstandingAjaxCalls`

Reply:
done